workspace/api/openfdd_bridge/device_poll_health.py

```
@@ -0,0 +1,360 @@
+            "samples": 0,
+            "valid_ratio": 0.0,
+            "stale": True,
+            "flips_per_day": 0.0,
+            "last_seen_age_min": None,
+        }
+    work = frame.loc[:, ["timestamp", column]].copy()
+    work["timestamp"] = pd.to_datetime(work["timestamp"], utc=True, errors="coerce")
+    work = work.dropna(subset=["timestamp"]).sort_values("timestamp")
+    vals = pd.to_numeric(work[column], errors="coerce")
+    valid = vals.notna()
+    samples = int(len(work))
```

Ruff (0.15.15)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)